modules/configuration/src/main/java/org/apache/ignite/internal/configuration/ConfigurationRegistry.java

```diff
@@ -245,12 +246,12 @@ public <T> T represent(List<String> path, ConfigurationVisitor<T> visitor) throw
         }
 
         if (node instanceof TraversableTreeNode) {
-            return ((TraversableTreeNode) node).accept(null, visitor);
+            return ((TraversableTreeNode) node).accept(null, null, visitor);
```

Comment:
I remember that we were discussing this place. "represent" must always pass a field, otherwise user may provide a path to password configuration and you won't obscure it.
`find` must return not only the object, but also a meta-information about it. This is the only way to do it. You also should add tests for such scenario.